afs/afs-mapdb-storage/src/main/java/com/powsybl/afs/mapdb/storage/MapDbAppStorage.java

```diff
@@ -680,7 +680,7 @@ public void addDependency(String nodeId, String name, String toNodeId) {
         if (dependencyNodes == null) {
             return Collections.emptySet();
         }
-        return dependencyNodes.stream().map(this::getNodeInfo).collect(Collectors.toSet());
+        return new LinkedHashSet<>(dependencyNodes.stream().map(this::getNodeInfo).collect(Collectors.toSet()));
```

Comment:
Use .collect(Collectors.toCollection(LinkedHashSet::new)) instead

Reply:
it was a mistake lo leave Collectors.toSet() . it should be normally Collectors.toList() for ordered list

Reply:
But it would break the API: to change later and sync everywhere